cli/data.py

```diff
@@ -167,6 +168,10 @@ def update(
     if print_stats:
         multiregion_dataset.print_stats("new_cases_and_deaths")
 
+    multiregion_dataset = nytimes_anomalies.filter_by_nyt_anomalies(multiregion_dataset)
+    if print_stats:
+        multiregion_dataset.print_stats("nytimes_anomalies")
```

Comment:
FWIW- I loosely based all of my code off the existing outlier detection code (directly below).